src/app/staking/components/delegates/DelegateCardList.tsx

```diff
@@ -101,6 +103,7 @@ export default function DelegateCardList({
             return (
               <DelegateCard
```

Comment:
I'm looking at the DelegateCard component you editing at `src/components/Delegates/DelegateCard/DelegateCard.tsx` and it only has `delegate` as a prop, I'm confused why this one accepts so many more props... weird. If it works it works though :)